styles/styles.css

```diff
@@ -0,0 +1,202 @@
+/* General Styling */
+html {
+  font-family: 'Nunito', Georgia, Times, serif;
+}
+
+h1, h2, h3, strong {
+	font-family: 'Ubuntu', Helvetica, Arial, sans-serif;
+}
+
+a {
+  color: #426642;
+}
+
+a:hover {
+  color: white;
+}
+
+.intro, .team, h1, h2 {
+  text-align: center;
+}
+
+ul {
+  list-style: none;
+}
+
+section:not(.intro) {
+  padding: 3em;
+}
+
+/* Fade in on page load animation
+Courtesy of: https://codepen.io/jmelgoza/pen/jEaGYg */
+
+/* make keyframes that tell the start state and the end state of our object */
+@-webkit-keyframes fadeIn { from { opacity:0; } to { opacity:1; } }
+@-moz-keyframes fadeIn { from { opacity:0; } to { opacity:1; } }
+@keyframes fadeIn { from { opacity:0; } to { opacity:1; } }
+
+h1 {
+  font-size: 3em;
+  margin-top: 3em;
+  opacity: 0;  /* make things invisible upon start */
+  -webkit-animation: fadeIn ease-in 1;  /* call our keyframe named fadeIn, use animattion ease-in and repeat it only 1 time */
+  -moz-animation: fadeIn ease-in 1;
+  animation: fadeIn ease-in 1;
+
+  -webkit-animation-fill-mode: forwards;  /* this makes sure that after animation is done we remain at the last keyframe value (opacity: 1)*/
+  -moz-animation-fill-mode: forwards;
+  animation-fill-mode: forwards;
+
+  -webkit-animation-duration: 1s;
+  -moz-animation-duration: 1s;
+  animation-duration: 3s;
+}
+
+/* Divider Line */
+.quote::after, .sponsors::after {
+  display: block;
+  content: "";
+  border-top: 1px dashed #426642;
+  margin-top: 2em;
+}
+
+/* Top Nav Bar*/
+nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: black;
+  color: whitesmoke;
+}
+
+nav ul {
+  display: grid;
+  flex-direction: row;
+  grid-template-columns: 5fr 1fr 1fr;
+}
+
+/* Intro Section */
+.intro {
+  display: flex
+  background-color: lightgrey;
+  padding: 2em;
+  /* Parallax Effect */
+  background-image: url("https://cdn.pixabay.com/photo/2016/09/30/03/39/pineapple-1704339_1280.jpg");
+  /* Set a specific height */
+  height: 500px; 
+  /* Create the parallax scrolling effect */
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+sub {
+  display: block;
+  margin-bottom: 3em;
+}
+
+.intro [href] {
+  width: 25%;
+  border: .25em solid #426642;
+  border-radius: 5px;
+  align: center;
+  padding: .5em;
+}
+
+/* Quote Section */
+.quote {
+  font-style: italic;
+}
+
+blockquote::before, blockquote::after {
```

Comment:
This is a good way to add the styling to the html without actually changing it. I'm definitely going to steal this!